### DISCUSSION TITLE
Résultat très loin de la réalité ?

### DISCUSSION THREAD

**Discussion by davidbgk** [cnumr/EcoIndex#215] [Bug]
Je fais un test sur cette page : https://larlet.fr/david/ (10 requêtes, 350Ko)

J'obtiens ce résultat : https://www.ecoindex.fr/resultat/?id=9dbfcdd9-ee4c-46da-8c4b-397fb4f6185b (126 requêtes, 1.7Mo)

Je n'arrive pas à comprendre ces écarts 🤔

**Comment by tbroyer** [comment]
EcoIndex scrolle jusqu'en bas de la page (https://www.ecoindex.fr/comment-ca-marche/#m%C3%A9thodologie-danalyse), ce qui génère les requêtes supplémentaires au fur et à mesure (apparemment une forme de preloading par instantpage.js)

**Reply by davidbgk** [reply]
Merci @tbroyer je comprends beaucoup mieux 👍 

Peut-être qu'il serait pertinent d'afficher le score avant/après le scroll pour rendre visible cette différence lorsqu'elle existe ? J'ai l'intuition que cela pourrait faire réfléchir à ce qui est chargé en asynchrone pour travailler dessus… en tout cas ça va être mon cas 😅

**Comment by yaaax** [comment]
C'est une bonne idée. À mettre en backlog quelque-part !
